src/index.ts

```diff
@@ -1265,25 +1372,31 @@ export abstract class PinejsClientCore<PinejsClient> {
 			method?: 'GET';
 			options: { $count: NonNullable<ODataOptions['$count']> };
 		},
+		overrides?: undefined,
 	): Promise<number>;
 	public request(
 		params: Params & { method?: 'GET'; id: NonNullable<Params['id']> },
+		overrides?: undefined,
 	): Promise<AnyObject | undefined>;
 	public request(
 		params: Omit<Params, 'id'> & { method?: 'GET' },
+		overrides?: undefined,
 	): Promise<AnyObject[]>;
 	public request(
 		params: Params & { method?: 'GET' },
+		overrides?: undefined,
 	): Promise<PromiseResultTypes>;
 	public request(
 		params: Params & {
 			method: 'PUT' | 'PATCH' | 'DELETE';
 		},
+		overrides?: undefined,
 	): Promise<void>;
 	public request(
 		params: Params & {
 			method: 'POST';
 		},
+		overrides?: undefined,
```

Comment:
All of these can be removed

Reply:
I was specifically referring to the `overrides?: undefined,` that were added, but it looks like you've left a bunch in and removed some other stuff - afaict the definition of the function should now be unchanged by this PR so the changes around here and above should all be reverted to match master

Reply:
I'm still in the middle of changes, but thanks for calling that out, because I did misunderstand what you meant.  They are in a separate commit because I was going to ask you about it.  I'll remove.